manifests/plugins/elasticsearch.pp

```diff
@@ -24,9 +25,13 @@
     content => template('collectd/plugins/elasticsearch/elasticsearch_collectd.py.erb'),
   }
 
-  collectd::plugins::plugin_common { 'elasticsearch':
-    package_name         => 'collectd-python',
-    plugin_file_name     => '20-elasticsearch.conf',
-    plugin_template_name => 'elasticsearch/20-elasticsearch.conf.erb',
+  collectd::plugin { 'elasticsearch':
+    package_name        => 'collectd-elasticsearch',
```

Comment:
this package doesn't exists and should either be removed or changed to collectd-python.  RHEL platforms fail on this.